pkg/receiver/jobreceiver/runner.go

```diff
@@ -0,0 +1,33 @@
+package jobreceiver
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/SumoLogic/sumologic-otel-collector/pkg/receiver/jobreceiver/builder"
+	"github.com/SumoLogic/sumologic-otel-collector/pkg/receiver/jobreceiver/output/consumer"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator"
+	"go.uber.org/zap"
+)
+
+// Build returns the job runner, the process responsible for scheduling and
+// running commands and piping their output to the output consumer.
+func (c Config) Build(logger *zap.SugaredLogger, out consumer.Interface) (builder.JobRunner, error) {
+	return &stubRunner{Consumer: out}, nil
```

Comment:
This is where we'd set up and return a real implementation that schedules & executes the jobs.